### PR TITLE
Added missing Oncotator optional arguments to CNV pair WDL.

### LINKED ISSUE #4319: CNV Somatic Pair Workflow (v4.0.1.0) not compatible with latest Cromwell
The CNV Somatic Pair workflow is missing workflow level inputs for "additional_args" and "mem_gb_for_oncotator" (both of which are fed into the Oncotator sub-workflow) and won't run on the latest Cromwell (v30.2).

### PR DESCRIPTION
**Pull request by samuelklee**
@ruchim @LeeTL1220 Can you confirm that this works before we merge and cut a release?  @droazen 

Closes #4319.

### CODE REVIEW

**Comment by samuelklee** [comment]
Also note that I decided to append `_for_oncotator` to `additional_args`, since this is sufficiently vague without the suffix.  However, analogous suffixes were not appended to exposed optional arguments for other tasks, since their names were less ambiguous.  This is the sort of grossness we can do away with once Cromwell handles this.

**Comment by ruchim** [comment]
LGTM :+1:

File: scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl [hunk 1]
```diff
@@ -111,8 +111,12 @@ workflow CNVSomaticPairWorkflow {
     #########################################
     Int? minimum_contig_length
 
-    # Ignored if not running oncotator
-    String oncotator_docker = "broadinstitute/oncotator:1.9.5.0-eval-gatk-protected"
```
[the remark this answers is not in the document]
Comment (samuelklee) [reply scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl]: In the Oncotator workflow itself.

**Comment by LeeTL1220** [comment]
All good then.  @samuelklee

**Pull request merged**